shared/notifications/index.js

```diff
@@ -0,0 +1,67 @@
+export const UI_NOTIFICATIONS = {
+  1: {
+    id: 1,
+    title: 'notifications#1Title',
+    description: 'notifications#1Description',
+    date: '02/22/2020',
+    image: 'images/swaps-logos-small.svg',
+    actionText: 'notifications#1ActionText',
+  },
+  2: {
+    id: 2,
+    title: 'notifications#2Title',
+    description: 'notifications#2Description',
+    date: '02/22/2020',
+    actionText: 'notifications#2ActionText',
+  },
+  3: {
+    id: 3,
+    title: 'notifications#3Title',
+    description: 'notifications#3Description',
+    date: '02/22/2020',
+    actionText: 'notifications#3ActionText',
+  },
+};
+
+/**
+* Messages and descriptions for all locale keys above are in app/_locales/en/messages.json.
+* For convenience, they are copied below. If you need to change them, you will have to edit
```

Comment:
Having to update these in two places does seem like the opposite of convenient :thinking:

Reply:
removed in f60d577dc